src/usr/share/gpgmailer/gpgmailmessage.py

```diff
@@ -94,6 +90,18 @@ def set_body(self, body):
         self._check_if_saved()
         self.message['body'] = body
 
+    # Adds a list of recipients.
+    #
+    # recipients: a list of one or more recipients for the message.
+    def set_recipients(self, recipients):
+        self._check_if_saved()
+        self.message['recipients'] = recipients
+
+    # Adds a list of gpg key fingerprints for recipients of the message.
```

Comment:
This comment should probably say what the list of fingerprints is used for. (Encryption.)